spec/routing/conferences_routing_spec.rb

```diff
@@ -10,19 +10,19 @@
       expect(get('/conferences/1')).to route_to('conferences#show', id: '1')
     end
 
-    it 'routes to #edit' do
+    xit 'routes to #edit' do
```

Comment:
Personally, I don't write routing spec for "normal", ie. restful controllers. I find them to be useful when you do something fancy with parameter parsing, routing contraints, or generally diverging from the expected pathname for a given controller.

The conferences controller is pretty standard. I'd have no problem with deleting these examples 😱